clients/java/src/main/java/io/camunda/client/api/response/EvaluateDecisionResponse.java

```diff
@@ -76,8 +76,12 @@ public interface EvaluateDecisionResponse {
    */
   String getTenantId();
 
+  /** Deprecated, please use {@link #getDecisionEvaluationKey()} instead. */
+  @Deprecated
+  long getDecisionInstanceKey();
```

Comment:
❓ the title says to deprecate the `decisionInstanceId` here we are deprecating the `decisionInstanceKey`. Is the title wrong?